<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>463</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>542</width>
    <height>463</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>542</width>
    <height>463</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Photoshop</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 40, 40);
color: white;</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QSpinBox{
	background-color:white;
	color: black;
}

QDoubleSpinBox{
	background-color:white;
	color: black;
}

QPushButton{
	background-color: rgb(61, 61, 61)
}</string>
   </property>
   <widget class="QLabel" name="label_img1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>321</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(25, 25, 25);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="vMirror">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>V Mirroring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hMirror">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>H Mirroring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grayScale">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>GrayScale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copy">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>410</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomOut">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoomIn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateR">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateL">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate Left</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sxValue">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>11</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1600000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="quant">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantization</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="toneValue">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="histogram">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalize">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Equalization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bright">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contrast">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Constrast</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="matching">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>180</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Histogram Matching</string>
    </property>
   </widget>
   <widget class="QPushButton" name="negative">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Negative</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="contrastValue">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="brightValue">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>-255</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="convolution">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply Convolution</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="syValue">
    <property name="geometry">
     <rect>
      <x>208</x>
      <y>11</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1600000</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kernelValue_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kernelValue_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>240</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kernelValue_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kernelValue_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kernelValue_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kernelValue_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kernelValue_8">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kernelValue_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="kernelValue_1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-255.000000000000000</double>
    </property>
    <property name="maximum">
     <double>255.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCarregar_Imagem_2"/>
    <addaction name="separator"/>
    <addaction name="actionSalvar_Imagem"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <action name="actionSalvar_Imagem">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionasda">
   <property name="text">
    <string>asda</string>
   </property>
  </action>
  <action name="actionCarregar_Imagem_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionCopy_Image">
   <property name="text">
    <string>Copy Image</string>
   </property>
  </action>
  <action name="actionCopy_Image_2">
   <property name="text">
    <string>Copy Image</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
